README.md

```diff
@@ -544,30 +544,26 @@ See http://tools.ietf.org/html/rfc3986#section-5
 
 ## `GuzzleHttp\Psr7\Uri::withQueryValue`
 
-`public static function withQueryValue(UriInterface $uri, $key, $value) -> UriInterface`
+`public static function withQueryValue(UriInterface $uri, $key, $value): UriInterface`
 
 Create a new URI with a specific query string value.
 
 Any existing query string values that exactly match the provided key are
 removed and replaced with the given key value pair.
 
-Note: this function will convert "=" to "%3D" and "&" to "%26".
```

Comment:
This shouldn't be part of the doc as it is logical to make it work and consistent with PSR-7.